kubernetes_state/metadata.csv

```diff
@@ -5,7 +5,7 @@ kubernetes_state.node.pods_capacity,gauge,,,,The total pod resources of the node
 kubernetes_state.node.cpu_allocatable,gauge,,cpu,,The CPU resources of a node that are available for scheduling,0,kubernetes,k8s_state.node.cpu_allocatable
 kubernetes_state.node.memory_allocatable,gauge,,byte,,The memory resources of a node that are available for scheduling,0,kubernetes,k8s_state.node.memory_allocatable
 kubernetes_state.node.pods_allocatable,gauge,,,,The pod resources of a node that are available for scheduling,0,kubernetes,k8s_state.node.pods_allocatable
-kubernetes_state.node.status,gauge,,,,The number of nodes in a given status, per tag,0,kubernetes,k8s_state.node.status
+kubernetes_state.node.status,gauge,,,,The number of nodes by schedulability; tagged 'status:schedulable' and 'status:unschedulable'.,0,kubernetes,k8s_state.node.status
```

Comment:
You get the number of nodes if you sum the metric over a cluster. Every node's metric is either 0 or 1.
This is the same as `docker.containers.running`: you get one timeseries per applicable tags, then sum it by host to get the total for the host.